edgexfoundry/device-virtual-go
# feat: Add MaxEventSize and ReadingUnits to configuration

Also add github.com/rcrowley/go-metrics to Attribution.txt
close #280

Signed-off-by: Ginny Guan <ginny@iotechsys.com>







**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/device-virtual-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] I have added unit tests for the new feature or bug fix (if not, why?) only add new configs
  <link to docs PR>
https://github.com/edgexfoundry/edgex-docs/pull/741 and https://github.com/edgexfoundry/edgex-docs/issues/726

## Testing Instructions

1. build device-virtual docker image from this branch and run with core services
2. update the configs and verify the responses
## New Dependency Instructions (If applicable)
